app/controllers/application_controller.rb

```diff
@@ -211,6 +222,7 @@ def load_html_blocks
   def authorize_collection
     # skip irrelevant cases
     return unless @collection
+    return if @collection.is_a?(DocumentSet)
```

Comment:
Question about rewriting line 228 - would that affect access if someone is using an old URL to access the document set?